executor/oci/resolvconf.go

```diff
@@ -2,10 +2,11 @@ package oci
 
 import (
 	"context"
+	"net/netip"
 	"os"
 	"path/filepath"
 
-	"github.com/docker/docker/libnetwork/resolvconf"
+	"github.com/moby/buildkit/executor/oci/internal/resolvconf"
```

Comment:
Not sure why we need this `resolvconf` package? Can we not just put content of this package directly here and make functions private?

Reply:
We usually try to avoid `internal` as enforced visibility in BuildKit but we can look at this as follow-up.